assets/package-lock.json

```diff
@@ -3484,18 +3484,26 @@
       "dev": true
     },
     "elliptic": {
-      "version": "6.5.3",
-      "resolved": "https://registry.npmjs.org/elliptic/-/elliptic-6.5.3.tgz",
-      "integrity": "sha512-IMqzv5wNQf+E6aHeIqATs0tOLeOTwj1QKbRcS3jBbYkl5oLAserA8yJTT7/VyHUYG91PRmPyeQDObKLPpeS4dw==",
+      "version": "6.5.4",
```

Comment:
Wish we could just remove this entirely; not like we use it ... https://www.npmjs.com/package/elliptic 🤷‍♂️